includes/sensei-functions.php

```diff
@@ -281,14 +281,48 @@ function sensei_user_login_url() {
  * duplicate of Sensei()->access_settings().
  *
  * @since 1.9.0
+ * @since 3.5.2 Added hook to filter the return.
  * @return bool
  */
 function sensei_is_login_required() {
+	global $post;
 
-	$login_required = isset( Sensei()->settings->settings['access_permission'] ) && ( true == Sensei()->settings->settings['access_permission'] );
+	$post_type = get_post_type( $post );
```

Comment:
This is unused, but see comment below.

Reply:
https://github.com/Automattic/sensei/pull/3631#discussion_r493021233